components/dashboard/GoodFirstIssues.tsx

```diff
@@ -45,18 +47,21 @@ export default function GoodFirstIssues({ issues }: GoodFirstIssuesProps) {
   const [selectedRepo, setSelectedRepo] = useState('All');
   const [selectedArea, setSelectedArea] = useState('All');
 
-  // Get current issues
+  const allIssues = issues;
 
-  let filteredIssues = issues;
 
-  const allIssues = issues;
+  const filteredIssues = useMemo(() => {
```

Comment:
I think instead of useMemo, we should use `useEffect` here as with react 18, `useMemo` also re-renders everything, without the change in dependency array.

Reply:
Ok